package.json

```diff
@@ -18,41 +18,52 @@
     }
   ],
   "scripts": {
-    "build": "bower install && gulp",
-    "jshint": "gulp jshint",
-    "jscs": "jscs gulpfile.js assets/scripts/*.js"
+    "build:production": "export SAGE_ENV=production && webpack",
+    "build": "export SAGE_ENV=development && webpack",
+    "watch": "export SAGE_ENV=development && node dev.js"
```

Comment:
That still wouldn't work on Windows. Then it would just say it can't find the command `SAGE_ENV=development`. Windows uses the `set` command to set environment variables, i.e., `set SAGE_ENV=development`